src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx

```diff
@@ -153,8 +169,39 @@ const DateFilter: FC<Props> = ({
         </MenuItem>
         <MenuItem onClick={() => setDatesFromOffset(365)}>Last Year</MenuItem>
       </Menu>
+      <DateChip
+        startDate={startDate}
+        endDate={endDate}
+        deleteDateFilter={() => setDatesFromRange(undefined, undefined)}
+      />
     </StyledFilterWrapper>
   );
 };
 
+interface DateChipProps {
+  startDate?: DateType;
+  endDate?: DateType;
+  deleteDateFilter: () => void;
+}
+
+function DateChip({
+  startDate,
+  endDate,
+  deleteDateFilter,
+}: DateChipProps): JSX.Element | null {
+  // DateChip should only display if we have at least one of startDate/endDate
+  if (!startDate && !endDate) return null;
+
+  // Get the date chip message. Structure varies if only one of the two dates.
+  // Might be worth extracting date message to a common helper func elsewhere?
+  const dateIntervalLabel = `${startDate || "Prior"} to ${endDate || "today"}`;
```

Comment:
to reflect the designs:
```suggestion
  const dateIntervalLabel = `${startDate || "Prior"} to ${endDate || "Today"}`;
```